src/qt/forms/debugwindow.ui

```diff
@@ -592,34 +592,17 @@
           <number>3</number>
          </property>
          <item>
-          <widget class="QPushButton" name="promptIcon">
-           <property name="enabled">
-            <bool>false</bool>
-           </property>
+          <widget class="GUIUtil::ThemedLabel" name="promptIcon">
```

Comment:
It appears that Qt Designer (at least version 5.12.8) does not behave well if a class name is prepended by a namespace -- it shows that the widget has a layout, that is obviously wrong.

Reply:
not applicable anymore since 2f23ad2